### PR TITLE
xlat: Move lpad/rpad to async

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

**Comment by jpereira** [comment]
It should be '%{foo: input1 inputN ...}

File: src/tests/keywords/pad [hunk 1]
```diff
@@ -9,35 +9,35 @@ update request {
 #
 #	rpad tests
 #
-if ("%{rpad:&Tmp-String-0 7}" != "test   ") {
+if ("%{rpad: 7 %{Tmp-String-0}}" != "test   ") {
```
Comment (mcnewton) [inline src/tests/keywords/pad]: Why change the argument order? Padding with spaces like this looks really weird (and to be technically correct, it should have two space at the start - the pad and the separator - which is even weirder).

**Comment by arr2036** [comment]
Just ignore it for now.  We need grouped xlat arguments first and probably an alternative syntax

**Comment by arr2036** [comment]
Will be addressed in https://trello.com/c/wHHX6MsF

**Pull request closed without merging**